front-end/src/pages/TeacherSubscriptions/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState, useContext } from 'react';
+
+// Context
+import { CurrentUserContext } from 'context/AppProvider';
+
+// Service
+import { getTeachersSubscription, deleteSubscription } from 'services/subscriptionService';
+
+// Components
+import SubscriptionsTable from 'components/common/SubscriptionsTable';
+import Loader from 'components/common/Loader/Loader';
+import NoSubscriptions from 'components/common/NoSubscriptions';
+
+// Styles
+import styles from './TeacherSubscriptions.module.scss';
+
+const TeacherSubscriptions = () => {
+  const { currentUser } = useContext(CurrentUserContext);
+  const [subscriptions, setSubscriptions] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const fetchSubscriptions = async (teacherId) => {
+    try {
+      setIsLoading(true);
+      const data = await getTeachersSubscription(teacherId);
+      setSubscriptions(data);
+      setIsLoading(false);
+    } catch (error) {
+      console.log(error);
+      setIsLoading(false);
+    }
+  };
+
+  const deleteSubscriptionById = async (id) => {
+    try {
+      setIsLoading(true);
+      const necessarySubscription = subscriptions.find((subscription) => {
+        return subscription._id === id;
+      });
+      const remove = await deleteSubscription(necessarySubscription._id);
```

Comment:
const remove, it wrong 
Please, change name

Reply:
fix it